apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -358,7 +374,9 @@ Func CameraPipe::build() {
         .unroll(c)
         .parallel(yo);
 
-    demosaiced_scheduler(processed);
+    demosaiced->intermed_compute_at.set(LoopLevel(processed, yi));
```

Comment:
I'd be OK with {processed, yi} as the idiomatic form we recommend. Making it variadic makes me a bit less happy, as we don't seem to have that pattern of pass-thru-ctor-args elsewhere. Plus we'd still need to be able to pass a whole LoopLevel as is.